### PR TITLE
Files for: ZEN22, ZEN32, ZEN54, ZEN71, ZEN72, ZEN73, ZEN74, ZEN76, ZEN77

### PR DESCRIPTION
**Pull request by TSHAdmin**

### CODE REVIEW

File: firmwares/zooz/ZEN72-800-LR.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"devices": [
+		{
+			"brand": "Zooz",
+			"model": "ZEN-numberhere",
```
Comment (kaechele) [inline firmwares/zooz/ZEN72-800-LR.json:5]: Should probably be ZEN72

File: firmwares/zooz/ZEN72-V02.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+	"devices": [
+		{
+			"brand": "Zooz",
+			"model": "ZEN-numberhere",
```
Comment (kaechele) [inline firmwares/zooz/ZEN72-V02.json:5]: Should probably be ZEN72

**Comment by TSHAdmin** [comment]
Hi Felix @kaechele, may i ask you to review the request? Thank you

**Comment by kaechele** [comment]
> Hi Felix @kaechele, may i ask you to review the request? Thank you

I could, but I'm not sure what weight it would carry, as I'm not directly affiliated with the project. I, for example, couldn't merge any changes.

**Comment by TSHAdmin** [comment]
Hi @AlCalzone may i ask you to review the request? Thank you

**Review by AlCalzone — APPROVED** [review]
I've removed the mention of the hardware revision from the changelogs, as this is likely to confuse users expecting a 1-click update without worrying about these details.

**Pull request merged**